docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx

```diff
@@ -229,7 +232,8 @@ export default function RootLayout({ children }) {
 
 <PagesOnly>
 
-To load Google Analytics for all routes, include the component directly in your custom `_app`:
+To load Google Analytics for all routes, include the component directly in your custom `_app` and
+pass in your measurement ID:
```

Comment:
Same question here: Should line 245 and 261 be changed from `GA-XYZ` to `G-XYZ`?

Reply:
Updated 👍